test/runner-unix.c

```diff
@@ -51,6 +53,67 @@ static void closefd(int fd) {
 }
 
 
+static int uv__cloexec(int fd, int set) {
+  int flags;
+  int r;
+
+  do
+    r = fcntl(fd, F_GETFD);
+  while (r == -1 && errno == EINTR);
+
+  if (r == -1)
+    return errno;
+
+  /* Bail out now if already set/clear. */
+  if (!!(r & FD_CLOEXEC) == !!set)
+    return 0;
+
+  if (set)
+    flags = r | FD_CLOEXEC;
+  else
+    flags = r & ~FD_CLOEXEC;
+
+  do
+    r = fcntl(fd, F_SETFD, flags);
+  while (r == -1 && errno == EINTR);
+
+  if (r)
+    return errno;
+
+  return 0;
+}
```

Comment:
Tangential: this is `uv__cloexec_fcntl()` from src/unix/core.c, right? We could simplify that to just a `F_SETFD` call because `FD_CLOEXEC` is the only defined flag anyway.